README.rst

```diff
@@ -9,7 +9,15 @@
  Rt - Python interface to Request Tracker API 
 ==============================================
 
-Python implementation of REST API described here: https://rt-wiki.bestpractical.com/wiki/REST
+Python interface to Request Tracker's REST API. With this library, you can
+automate most actions with Request Tracker that you would be able to make
+with the web interface.
+
+ * Version 2.0 of the API must be installed as an extension, but is more
```

Comment:
V2 ships with RT5, which has seen a stable release a couple of months ago.

Reply:
ah right, I should add this subtility in the README. for 4.x the API needs to be installed as a plugin, but it is true that it's builtin to 5.x (along with the Token auth plugin) so no additional plugin is required.